mlir/tools/mlir-tblgen/AttrOrTypeDefGen.cpp

```diff
@@ -707,7 +709,7 @@ class DefGenerator {
   void emitParsePrintDispatch(ArrayRef<AttrOrTypeDef> defs);
 
   /// The set of def records to emit.
-  std::vector<const llvm::Record *> defRecords;
+  std::vector<const Record *> defRecords;
```

Comment:
BTW, why do we use `std::vector` here instead of `SmallVector`. Is this intentional?

Reply:
I am not really sure.

Reply:
There isn't a specific reason why to use std::vector over SmallVector here.